CMakeLists_IOS.txt

```diff
@@ -0,0 +1,206 @@
+# This file is based off of the https://code.google.com/archive/p/ios-cmake/
```

Comment:
If the licenses are compatible, and you changed the code, you should add, eg: Copyright 2017 the Monero project. Or maybe Portions copyright 2017 the Monero project. The main copyright notice stays, you just append. It is much simpler to licence the modifications under the same licence as the original.